### ISSUE TITLE
Newcomers Guide does not link to any style guidelines

### ISSUE THREAD

**Issue by jrrickerson** [coala/coala#4255]
For new contributors, the Newcomer's guide to fixing your first issues and submitting back to the project does not include a style guide.  The Newcomer's guide should link to the Writing_Good_Commits.rst at least.  Consider also including links to PEP8 and Writing_Documentation.rst since newcomer issues are often documentation fixes.

**Comment by IpshitaC** [comment]
Hi @jrrickerson ! Will it be a good idea to add the links in Step 3 : Picking up a New Issue section of the Newcomer's guide, as a note?

**Comment by yash-nisar** [comment]
We do have a link to `Writing_Good_Commits.rst` : 
> Before starting to write your first commit, check out this link: Writing good commits.

We have also included a link to the Newcomer issues (which include documentation typos too) : 
> Now it is time to pick an issue. It is the best way to familiarise yourself with the codebase. Here is the link that will lead you to Newcomers issues.

Regarding the style guide, IMO, I don't think we need the user to do an indepth analysis about `PEP8` as running `coala` takes care of those issues, nor is the knowledge of locally building and testing documentation required. These newcomer issues are simply typos, changing variable names, etc which are meant to get the user accustomed to the workflow at coala. We mainly focus on the 'git handling' part here.
